src/components/custom/_dtrading.tsx

```diff
@@ -1,12 +1,12 @@
-import React from 'react'
+import React, { ReactElement } from 'react'
 import styled from 'styled-components'
 import { graphql, useStaticQuery } from 'gatsby'
 import device from 'themes/device'
 import { Container, SectionContainer } from 'components/containers'
 import { Header, Text, QueryImage } from 'components/elements'
 
 type TradingType = {
-    title?: string
+    title?: string | ReactElement
```

Comment:
What is the reason of having ReactElement as a type for title ?

Reply:
We are using **Localize** component in some of the places so we need ReactElement type here